utilities/CovidDownloader.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+from urllib.request import urlopen
+import json
+import csv
+
+
+class csvDownloader():
+	def __init__(self):
+		pass	
+
+	def find_data(self, countryName):
+		
+
+		df = pd.read_csv(url)
+		global myDict 
+		myDict = df.to_dict()
+
+		global x
+		x = list(myDict['Name'].keys())[list(myDict['Name'].values()).index(countryName)]
+		
+		message = "\n"
+		for value in myDict:
+			
+			message += str(f"{value}: {myDict[f'{value}'][x]} \n")
+		
+		return str(message)
+
+	def country_list(self):
+		df = pd.read_csv(url)
+		global myDict 
+		myDict = df.to_dict()
+
+		clist = []
+		for value in myDict['Name'].values():
+			clist.append(str(f"{value}"))
+		
+		sorted_clist = sorted(clist, key=str.lower)
+		
+		
+		return sorted_clist
```

Comment:
we can have Pandas read only the first column of the csv. We can also use the to_list() rather than creating and appending one ourselves. So these lines can probably be simplified to the following:
```
# the first column of covid data is the country names, titled "Name"
df = pd.read_csv(url, usecols=[0])
return sorted(df.Name.to_list(), key=str.lower)
```